dockers/docker-dhcp-server/Dockerfile.j2

```diff
@@ -13,6 +14,7 @@ ENV IMAGE_VERSION=$image_version
 RUN apt-get update &&       \
     apt-get install -f -y   \
         tcpdump             \
+        python3-dev         \
```

Comment:
why need this one, this one will increase the disk size, we should remove after done.

Reply:
`RUN pip3 install psutil` would fail in mavell platform without this deb. Added `apt-get remove` for this dev after pip install. Pls refer to this PR: https://github.com/sonic-net/sonic-buildimage/pull/16560/files#diff-0abb3d0364d7da24f9d28b3d724a5a73aefabaef11fcdc1284da806ad8604d3a